pyart/retrieve/kdp_proc.py

```diff
@@ -21,11 +33,994 @@
 
 import time
 import numpy as np
+from functools import partial
+
+# Constants in the Kalman filter retrieval method (generally no need to modify them)
+SCALERS = [0.1,10**(-0.8),10**(-0.6), 10**(-0.4), 10**(-0.2), 1,
+           10**(0.2), 10**(0.4), 10**(0.6), 10**(0.8),1, 10]
+           
+PADDING = 50 # Noise padding of the psidp signal (before and after signal)
+SHIFT = 13 # Shifting of the final signal
+
+from scipy import optimize, stats, interpolate, linalg, signal
+from pyart.retrieve import _kdp_proc
+from pyart.config import get_field_name, get_metadata, get_fillvalue
+
+def kdp_schneebeli(radar, gatefilter=None, fill_value = None, psidp_field=None,
+                kdp_field=None, phidp_field=None, band = 'C', rcov = 0, pcov = 0,
+                prefilter_psidp = False, filter_opt = {}, parallel = True):
```

Comment:
The default for filter_opt should be None with the empty dictionary generated in the function is None is detected.  [Mutable default options are dangerous.](https://python-guide.readthedocs.io/en/latest/writing/gotchas/#mutable-default-arguments)